sqlglot/dialects/bigquery.py

```diff
@@ -168,6 +170,7 @@ class Generator(generator.Generator):
             exp.DatetimeAdd: _date_add_sql("DATETIME", "ADD"),
             exp.DatetimeSub: _date_add_sql("DATETIME", "SUB"),
             exp.DateDiff: lambda self, e: f"DATE_DIFF({self.sql(e, 'this')}, {self.sql(e, 'expression')}, {self.sql(e.args.get('unit', 'DAY'))})",
+            exp.GroupConcat: rename_func("STRING_AGG"),
```

Comment:
can you also add the inverse so that string_agg can be read in as a group_concat? line 139